cplusplus/include/vips/VImage8.h

```diff
@@ -1055,6 +1056,24 @@ class VImage : public VObject
 		return( result ); 
 	}
 
+	/**
+	 * Acquire VRegion covering the whole image.
+	 */
```

Comment:
This will make a region with no left/top/width/height set yet, rather than one that covers everything (that would need a huge amount of memory!).